test/integration_test.go

```diff
@@ -202,7 +216,7 @@ func TestIntegration_ManyVotesScenario(t *testing.T) {
 	require.NoError(t, err)
 
 	// ##### DECRYPT BALLOTS #####
-	time.Sleep(time.Second * 1)
+	time.Sleep(time.Second * 10)
```

Comment:
Normally a computer can crunch tons of processing in a second. It would be interesting to benchmark things.

Reply:
The beauty of consensus... 😁 This is not only purely about executing opcodes, but there are nodes that need to process transaction, wait some time to gather them, check if the leader is acting correctly, propose to elect a new leader, revert back because the leader woke up, etc... All this takes some time, and it is hard to predict it precisely. And the more we add nodes and transaction, the more we increase the incertitude.